project/Settings.scala

```diff
@@ -273,14 +273,14 @@ object Settings {
     /** https://github.com/lucidworks/spark-solr/issues/179 */
     dependencyOverrides ++= {
       val deps = Seq(
-        "com.fasterxml.jackson.core" % "jackson-core" % "2.6.5",
-        "com.fasterxml.jackson.core" % "jackson-databind" % "2.6.5",
-        "com.fasterxml.jackson.core" % "jackson-annotations" % "2.6.5"
+        "com.fasterxml.jackson.core" % "jackson-core" % "2.6.7",
```

Comment:
This update needs to be double checked on EMR cluster; see https://github.com/lucidworks/spark-solr/issues/179